### ISSUE TITLE
Narrator reading order for About dialog appears incorrect

### ISSUE THREAD

**Issue by miniksa** [microsoft/terminal#11295]
This is the order:

![image](https://user-images.githubusercontent.com/18221333/134210932-753323e3-95cb-4b2f-8e65-4ce69a116f6f.png)

I think it should be reading the text at the top first before reading the "Documentation link" one below it.

**Comment by zadjii-msft** [comment]
Note: This still repros after the addition of the "Source Code" link. I have no idea why the `TextBlock` would get read after the first `HyperlinkButton`. Maybe because tab-focus lands there first, so narrator reads it first, then the rest of the items? \</conjecture>

**Comment by zadjii-msft** [comment]
Op, looks like this is a WinUI bug: https://github.com/microsoft/microsoft-ui-xaml/issues/5909

(admittedly, that one was closed for being stale)
